Programmers/같은 숫자는 싫어/seungbin.py

```diff
@@ -0,0 +1,15 @@
+def solution(arr):
+    ans = []
+    ans.append(arr[0]) #처음은 중복X
+    x=arr[0] #비교 위해 저장
+    
+    for i in arr:
+        #앞과 다르면 추가
+        if i != x:
+            ans.append(i)
+            print(i)
+        else:
+            continue
+        x=i #x값 초기화
```

Comment:
else 값을 사용하지 않고 if 문 안에서 x를 초기화해도 가능할 것 같아요!

그리고 x를 두지 않고 ans[-1] 값과 비교하는 방법도 좋을 것 같습니다 ~~!

```suggestion
    for i in arr:
        #앞과 다르면 추가
        if i != x:
            ans.append(i)
            x=i #x값 초기화   
```